### ISSUE TITLE
Please add the ability to open a new tab after the current one and not the last one

### ISSUE THREAD

**Issue by Nordvis** [kiwibrowser/src.next#562]
Hello! I understand that this question may have already written, but I appeal to the developers - please add the ability to open a new tab immediately after the current one and not the most recent! Is it so difficult to implement?  If it were not for this annoying feature, Kiwi would have been a really great browser. As far as I understand I am not the only one who asked for this option. I really hope that the developers will hear our requests. Thanks!

**Comment by netrunner-exe** [comment]
Yes, I'm also waiting for the opportunity to choose this ... it would be much better if the developers added this option when the new tab opens at the end is very inconvenient
